__tests__/app.test.js

```diff
@@ -163,10 +150,60 @@ describe('app', () => {
               })  
             })
 
-        })
+            })
+            
+            describe('GET/api/articles/:article_id/comments', () => {
+                it('status 200 - successful request should return the correct comment when passed an article_id', () => {
+                    return request(app)
+                        .get('/api/articles/1/comments')
+                        .expect(200)
+                        .then(({ body }) => {
+                            console.log(body)
+                            expect(body.comments).toBeInstanceOf(Array)
+                            expect(body.comments).toHaveLength(11)
+                            body.comments.forEach((comment) => {
+                                expect(comment).toMatchObject({
+                                    comment_id: expect.any(Number),
+                                    votes: expect.any(Number),
+                                    created_at: expect.any(String),
+                                    author: expect.any(String),
+                                    body: expect.any(String),
+                                    article_id: expect.any(Number)
+
 
-       
+                                })
+
+                            })
+                        
 
+                    })
+                });
+
```

Comment:
It would be good to test the `order` that the comments come back in. You can use the npm package `jest-sorted` to do this or do it in some other way.